tests/test_cli.py

```diff
@@ -0,0 +1,92 @@
+import os
+from pathlib import Path
+from sabres.cli import main
+from sabres import merge_sabres
+import pytest
+
+
+
+@pytest.mark.parametrize(
+    "options,expected", 
+    [
+        (["--help"], "show this help message and exit"),
+        (["-h"], "show this help message and exit"),
+        (["--version"], "SABRes v"),
+])
+def test_helpers(capsys, options, expected):
```

Comment:
testing log/print output doesn't really add much as its just testing argparse is working. Theres little chance for a "bug" to appear in this code and now you have two places to change CLI messages.